src/Controller/EnMarche/CitizenInitiativeController.php

```diff
@@ -197,6 +199,41 @@ public function attendAction(Request $request, CitizenInitiative $initiative): R
     }
 
     /**
+     * @Route("/{uuid}/{slug}/partage-au-comite", requirements={"uuid": "%pattern_uuid%"}, name="app_citizen_initiative_committee_share")
+     * @Method("GET|POST")
+     * @Security("is_granted('ROLE_HOST')")
+     */
+    public function shareToCommitteeAction(Request $request, CitizenInitiative $initiative)
+    {
+        $this->disableInProduction();
+
+        if (empty($committees = $this->get('app.committee.manager')->getAdherentCommitteesHost($this->getUser()))) {
```

Comment:
As I remember, an adherent can share in the committee where he is supervisor (AL) and not just host (co-AL), can't he ?